src/styles/app.styl

```diff
@@ -36,6 +37,18 @@ body
     mix-blend-mode multiply
     align-items center
     overflow-y auto
+    .corner
+        position relative
+        align-self flex-end
+        margin rem(12) 2rem rem(-44) 0
```

Comment:
-44 sounds weird

Reply:
This is a negative margin to place the corner as expected in the spec. Why is this weird ?

Reply:
Peut-être pour la valeur qui n'est pas un multiple de 16. Sinon peut-être parce que généralement on évite les margin négative pour corriger un souci, on essaye plutôt de modifier le parent.